model_cards/mrm8488/RuPERTa-base/README.md

```diff
@@ -0,0 +1,5 @@
+---
+language: spanish
+widget:
+- text: "España es un país my <mask> en UE"
```

Comment:
```suggestion
```

Reply:
Feel free to change it if you want. In fact I made a typo: I meant "España es un país muy <mask> en UE", not "my"